.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+### Description
+
+Explain what you did, what you expected to happen, and what actually happens.
+
+### Reproduction Steps and Sample Code
+
+Try to reproduce your bug on https://snack.expo.io/ and provide a link.
+If you can't reproduce the bug on Snack, provide a sample project. At the very least, provide an example of your code.
+
+### Solution
+
+What needs to be done to address this issue? Ideally, provide a pull request with a fix.
+
+### Additional Information
+
+* React Native version: [FILL THIS OUT: Be specific, filling out "latest" here is not enough.]
+* Platform: [FILL THIS OUT: iOS, Android, or both?]
+* Development Operating System: [FILL THIS OUT: Are you developing on MacOS, Linux, or Windows?]
```

Comment:
`Development Operating System` and `Dev Tools` are not really useful for us to debug. Thoughts @Monte9